src/PsfFlux.cc

```diff
@@ -94,7 +94,12 @@ void PsfFluxAlgorithm::measure(
         ) {
             badBits |= exposure.getMaskedImage().getMask()->getPlaneBitMask(*i);
         }
-        fitRegion.intersectMask(*exposure.getMaskedImage().getMask(), badBits);
+        auto tmpSpans = afw::geom::SpanSet::fromMask(*exposure.getMaskedImage().getMask(),
+                                                     [&badBits]
+                                                     (afw::image::MaskPixel const & bitPattern)
+                                                     {return bitPattern & badBits;});
+        fitRegion.setSpans(fitRegion.getSpans()->intersectNot(*tmpSpans)->clippedTo(
+            exposure.getMaskedImage().getMask()->getBBox()));
```

Comment:
If you think an API update is worth it, please file a ticket now so we don't forget. If you suspect it will likely be unneeded, don't bother with the ticket.